process-migration-service/pom.xml

```diff
@@ -23,6 +23,7 @@
     <frontend-maven-plugin.version>1.7.6</frontend-maven-plugin.version>
     <maven-resources-plugin.version>3.1.0</maven-resources-plugin.version>
     <node.version>v11.6.0</node.version>
+    <npm.version>6.9.0</npm.version>
```

Comment:
For this to have an effect, you have to put `npmVersion` property with the value of this property in the frontend-maven-plugin configuration.